src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpAppealsController.cs

```diff
@@ -199,6 +202,46 @@ public async Task<IActionResult> AppealUnsuccessful(Guid applicationId)
             return View("~/Views/Appeals/AppealUnsuccessful.cshtml", model);
         }
 
+        [HttpGet("application/{applicationId}/appeal/successful")]
+        public async Task<IActionResult> AppealSuccessful(Guid applicationId)
+        {
+            var appeal = await _appealsApiClient.GetAppeal(applicationId);
+            if (appeal?.Status != AppealStatus.Successful && appeal?.Status != AppealStatus.SuccessfulFitnessForFunding && appeal?.Status != AppealStatus.SuccessfulAlreadyActive)
```

Comment:
might need to change this so it has an **appeal is null** check as I think line 220 is susceptible to a NullReferenceException

Reply:
ta